src/openzaak/components/zaken/tests/test_rol.py

```diff
@@ -547,6 +547,51 @@ def test_create_rol_with_contactpersoon(self):
         self.assertEqual(rol.contactpersoon_rol_naam, "test name")
         self.assertEqual(rol.contactpersoon_rol_telefoonnummer, "061234567890")
 
+    def test_create_rol_with_empty_aoaIdentificatie(self):
+        url = get_operation_url("rol_create")
+        zaak = ZaakFactory.create()
+        zaak_url = get_operation_url("zaak_read", uuid=zaak.uuid)
+        roltype = RolTypeFactory.create(zaaktype=zaak.zaaktype)
+        roltype_url = reverse(roltype)
+        data = {
+            "zaak": f"http://testserver{zaak_url}",
+            "betrokkene_type": RolTypes.natuurlijk_persoon,
+            "roltype": f"http://testserver{roltype_url}",
+            "roltoelichting": "awerw",
+            "betrokkeneIdentificatie": {
+                "anpIdentificatie": "12345",
+                "verblijfsadres": {
+                    "wplWoonplaatsNaam": "test city",
+                    "gorOpenbareRuimteNaam": "test",
+                    "aoaPostcode": "1111",
+                    "aoaHuisnummer": 1,
+                },
+                "subVerblijfBuitenland": {
+                    "lndLandcode": "UK",
+                    "lndLandnaam": "United Kingdom",
+                    "subAdresBuitenland_1": "some uk adres",
+                },
+            },
+        }
+
+        # field is still required
+        response = self.client.post(url, data)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        validation_error = response.json()["invalidParams"][0]
+        self.assertEqual(
+            validation_error["name"],
+            "betrokkeneIdentificatie.verblijfsadres.aoaIdentificatie",
+        )
+        self.assertEqual(validation_error["code"], "required")
```

Comment:
We can use the util from `vng_api_common` for this, it's already imported in this file

```suggestion
        validation_error = get_validation_errors(
            response, "betrokkeneIdentificatie.verblijfsadres.aoaIdentificatie"
        )
        self.assertEqual(validation_error["code"], "required")
```